### ISSUE TITLE
missing=None with FileUploadWidget

### ISSUE THREAD

**Issue by TomOudeNijhuis** [Pylons/deform#374]
When using a SchemaNode like:
```
filedata = colander.SchemaNode(
    deform.FileData(), missing=None,
    widget=deform.widget.FileUploadWidget(FileStorage),
    title='Bestand')
```
I get the a _Invalid pstruct: {'upload': ' is not a FieldStorage instance'}_ error when leaving the field empty. Non-empty fields work fine.
Replacing `None` with `b''` or `''` for the missing parameter gives the same result.
I am using version 2.0.5

**Comment by stevepiercy** [comment]
Would `missing= colander.drop` satisfy your requirement? See https://docs.pylonsproject.org/projects/colander/en/latest/api.html#schema-related

**Comment by TomOudeNijhuis** [comment]
That would satisfy me, but unfortunately using `colander.drop` or `colander.none` as missing value results in the same error.

**Comment by stevepiercy** [comment]
Thanks for verifying. I've flagged this issue as a bug. If you would like to submit a pull request and tests to fix the issue, I'm fairly sure it would be accepted.

**Comment by stevepiercy** [comment]
This should have been resolved in #360. Closing. If not, please reopen or create a new issue.
